base/src/main/scala/co/uproot/abandon/Process.scala

```diff
@@ -225,6 +225,8 @@ object Processor {
 
       if (postsNoAmount.length > 1) {
         throw new InputPosError(s"More than one account posted with unspecified amount: ${postsNoAmount.map(_.accName).mkString(",")}", tx.pos)
+      } else if (postsWithAmount.isEmpty) {
+        throw new InputPosError(s"Transaction with single posting: ${tx.posts.head.accName}", tx.pos)
```

Comment:
I like the check you added (`postsWithAmount.isEmpty`). It is more general than I had imagined.

However, the error messages seems to be too narrow. How about: "Transaction with unspecified amount"?